modules/simulator.js

```diff
@@ -72,8 +77,36 @@ function createOrganism(land)
     var randomMass = Constant.MIN_MASS + (Math.random()*Math.random()) * Constant.MAX_MASS;
     var randomMinSpeed =  (Math.random()) + Constant.MIN_SPEED;
 
-    // create Organism(generation, mass, energy, x, y, vision_angle, vision_range, minSpeed, fertility)
-    return new Organism(1, randomMass, randomX, randomY, 100, 100, randomMinSpeed, 1);
+    // 
+    if(organismCounter%3 == 0) {
```

Comment:
no need for %3 when you're using getRandomInt(0,4) but i think a switch statement might look better